### ISSUE TITLE
Getting error of missing closing DIV

### ISSUE THREAD

**Issue by thonal** [andrewscaya/mod_simpleemailform#18]
Hi,

When I tried to validate a website, it failed on the email form saying it was missing a closing div. After some digging I noticed that the missing closing div could be resolved by adding it to line 1587 of 
"/modules/mod_simpleemailform/sefv2.modsimpleemailform.php" :

```
        .......$this->output .= "</table>\n";

        $this->output .= "</form>\n";

# FLS: Added div. error from validator that is was missing!
        $this->output .= "</div>\n";

        if ($this->formTestMode === 'Y') {
            $this->output = htmlspecialchars($thi.....

```
I don't know if this is a correct 'fix' or a hack to make it work? It does seem that it closes the div opened at line 1529:
```
        // Present the Email Form.
        $this->output .= !empty($this->formCssClass)
            ? "<div class=\"" . $this->formCssClass . "\">\n"
            : '';

```

Could you please advise?

Regards,

Thonal

**Comment by andrewscaya** [comment]
Hello Thonal,

You are absolutely right!  The missing 'div' will be added immediately on the develop branch and will be part of the next release scheduled to be published in August 2017.

Many thanks!

Best regards,

Andrew

**Comment by andrewscaya** [comment]
Bug fix completed - commit #bc977a9.
